target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -217,7 +217,8 @@ case "$FIRMWARE" in
 	tplink,archer-c59-v2|\
 	tplink,archer-c60-v1|\
 	tplink,archer-c60-v2|\
-	tplink,archer-c6-v2)
+	tplink,archer-c6-v2|\
+	tplink,archer-a6-v2)
```

Comment:
Apply correct sorting, please.

Reply:
back to c6-v2-us, so it is fine to keep in same position?